src/server.rs

```diff
@@ -218,7 +219,9 @@ impl ValidatorService for Validator {
             .into_iter()
             .map(|message| TxOut {
                 value: Amount::ZERO,
-                script_pubkey: message.into(),
+                script_pubkey: message
+                    .try_into()
+                    .expect("Failed to convert coinbase message to script"),
```

Comment:
Are you sure that you want to panic here, instead of returning a status error to the client?